pwiz_tools/Shared/Common/Chemistry/MzTolerance.cs

```diff
@@ -60,16 +63,32 @@ public MzTolerance(double value = 0, Units units = Units.mz)
             return 0;
         }
 
+        public static implicit operator MzTolerance(double tolerance)
```

Comment:
I don't think this implicit conversion should exist, because it's inherently ambiguous to anyone looking at the code whether it's going to be mz or ppm.
How about two static methods:
`public static MzTolerance FromPpm(double ppm)
`and
`public static MzTolerance FromMz(double mz)`

Reply:
It is used to reduce the amount of code changes. There is a lot of places in the code, especially in tests, where tolerance is passed to a method as a single number. With this operator in place it is implicitly converted to mz tolerance without need for explicit conversion.